### ISSUE TITLE
Device frame like emulators

### ISSUE THREAD

**Issue by rajeshjeshar** [Genymobile/scrcpy#1491]
If the window of the screen comes with a mobile device frame,that will give a better attraction than the normal window border
[Reference ](https://image.winudf.com/v2/image1/Y29tLmFuZHJvaWRzcmMuYXJ0Z2VuZXJhdGVyX3NjcmVlbl8xXzE1NTI4ODAxMjBfMDgz/screen-1.jpg?fakeurl=1&type=.jpg)

**Comment by TheRafaelFarias** [comment]
I agree, will be much better and beautiful to emulate the phone screen

**Comment by soul4soul** [comment]
I'd like to see this option too, it would help with app development on devices that don't have square screens. I was able to make a frame for the emulator in android studio but often times I connect to the actual device. Being able to apply the frame over scrcpy would be great.

**Comment by rom1v** [comment]
Refs #457

**Comment by Mrithul-E** [comment]
any updates

**Comment by kevin-roan** [comment]
Yes something that iOS device frame would be better. Or a hyphotetical device frame with even bessels or something
